### DISCUSSION TITLE
Define model fields metadata, and access it in JS

### DISCUSSION THREAD

**Discussion by krzys1u** [prisma/prisma#6804] [Q&A] [answered]
Is it possible to get my models column names and types in JS define custom metadata (or tags) for each column (field) ?

I want to create autogenerated admin panel for prisma schema, and i need possibility to read schema metadata in JS to render forms and lists of objects, additionally i need possibility to define some metadata in schema to read it in JS and customize my forms and lists

**Comment by mbrowne** [comment, marked as answer]
I'm not sure about custom metadata, but I found this answer that might be helpful:
https://github.com/prisma/prisma/discussions/4425#discussioncomment-143293

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
